src/main/scala/com/microsoft/hyperspace/index/rules/JoinIndexRule.scala

```diff
@@ -359,27 +375,6 @@ object JoinIndexRule
     compatibleIndexPairs.map(indexPairs => JoinIndexRanker.rank(indexPairs).head)
   }
 
-  /**
-   * Returns list of column names which must be in the indexed columns config of a selected
-   * index.
-   *
-   * @param plan logical plan
-   * @param condition join condition used to pull out required indexed columns in the plan
-   * @return list of column names from this plan which must be part of indexed columns in a chosen
-   *         index
-   */
-  private def requiredIndexedCols(plan: LogicalPlan, condition: Expression): Seq[String] = {
-    val cleanedPlan = CleanupAliases(plan)
```

Comment:
no need to clean up aliases now?

Reply:
Removed it for https://github.com/microsoft/hyperspace/issues/25. Let me revert alias related changes and take it in a separate PR

Reply:
Sounds good. Let's minimize the scope for this PR.

Reply:
based on other discussions and closed PR #102 , aliases aren't supported in Join rule. So keeping this change as discussed